Start of a new tutorial: FAIR-ego. DRAFT (#4351)
galaxyproject/training-material: topics/fair/tutorials/fair-ego/tutorial.md

> <agenda-title></agenda-title>
>
> In this tutorial, we will cover:
>
> 1. TOC
> {:toc}
>
{: .agenda}

# Excercise 1: Find the experiment

## The Problem

Imagine the following situation: An undergrad student wants to do a lab-based project with you. This is great because your post-doc has recently left the lab and there are several promising leads from their work that can be followed up. You propose the following topic to the student: 

**Effect of mutations in the gene CHES-I-like on feeding behaviour in _Drosophila melanogaster_**

The only problem is, you can't remember what your post-doc has done so far. This is no problem, as you have followed your data management plan to the letter: _"lab books will be preserved for 10 years in the PI’s office"._ Let's see how effective this strategy actually is.

<question-title>1</question-title>
The PDF below contains a scan of the lab books in this example. Somewhere, there is an experiment to show the effect of the knock-down of a gene called CHES-I-like on the feeding behaviour of fruit flies. Try to find this information in the lab book.

> > [5 years worth of lab books](Lab-book_excercise1.pdf)

Reviewer: this is actually hilarious, I love this exercise.

> > 12. Staining protocol
> > 13. Which microscope
> > 14. Which laser lines
> > 15. What exposure time
> > 16. Filter settings
> > 17. Magnification
> > 18. Pinhole size
> > 19. Number of z slices
> > 20. ...
> > 
> {: .solution}
>
{: .question}

## The solution

This excercise illustrates how much information you need to keep to be able to reproduce experiments, be it your own data or that of other scientists.
More often than not, this level of detail is not recorded / published, which may explain part of the **reproducibility crisis** {% cite Baker2016 %}.

This information about the data is called **metadata** and is arguably as important as the data itself.  

# Excercise 3: Get the data

## The Problem

Imagine the following situation: it has been a slow couple of years in terms of publications and grant income. You really need to get papers out, but you are short of research money. No worries! There is tons of data pit there for you to re-analyse, interpret, and integrate with your own data. You have found a particularly useful study in a paper and you want to use the data to get your research going again. In the data sharing statement, you find the following sentence: 

> _**The data that support the findings of this study are available from the corresponding author upon reasonable request.**_

Reviewer: ```suggestion
> The data that support the findings of this study are available from the corresponding author upon reasonable request.
{: .quote}
```

probably a good use for a 'quote'

---
layout: tutorial_hands_on
title: FAIR and RDM from a selfish perspective
abbreviations:
  FAIR: Findable, Accessible, Interoperable, Reusable
  GTN: Galaxy Training Network
  RDM: Research Data Management
  DPM: Data Management Plan
level: Introductory
zenodo_link: ''
questions:
- Why should other researchers adhere to FAIR principles?
- How can I profit from good RDM?
objectives:
- Learn about the benefits of good data management
time_estimation: "30M"
key_points:
- RDM can lead save time and money.
tags:
- fair
- gtn
- rdm
contributions:
  authorship:
    - korneelhens
    - nomadscientist
  funding:
    - elixir-fair-data
subtopic: fair-data

requirements:
  - type: "internal"
    topic_name: fair
    tutorials:
      - fair-intro
   
draft: true

---

# Introduction

> **You** are obviously too important to bother with this research data management ({RDM}) nonsense. Just copy the data management plan ({DPM})from a previous grant application and stop wasting time.

Reviewer: ```suggestion
**You** are obviously too important to bother with this {RDM} nonsense. Just copy the {DMP} from a previous grant application and stop wasting time.
```

you can just use `{..}` and it'll properly expand it also the first time :) (also this is a funny introduction, it's nice! :D)

However, there might be some value in persuading your colleagues to do their {RDM} properly and to adhere to {FAIR} standards. In this tutorial, we explore the purely selfish reasons for encouraging good data management practices based on some real-life examples.

> <agenda-title></agenda-title>
>
> In this tutorial, we will cover:
>
> 1. TOC
> {:toc}
>
{: .agenda}

# Excercise 1: Find the experiment

## The Problem

Imagine the following situation: An undergrad student wants to do a lab-based project with you. This is great because your post-doc has recently left the lab and there are several promising leads from their work that can be followed up. You propose the following topic to the student: 

> **Effect of mutations in the gene CHES-I-like on feeding behaviour in _Drosophila melanogaster_**

Reviewer: ```suggestion
> Effect of mutations in the gene CHES-I-like on feeding behaviour in _Drosophila melanogaster_
{: .quote}
```

we try and avoid bold text in these situations so maybe a quote gives the same feeling of "this sets apart the text"

The only problem is, you can't remember what your post-doc has done so far. This is no problem, as you have followed your data management plan to the letter: _"lab books will be preserved for 10 years in the PI’s office"._ Let's see how effective this strategy actually is.

> <question-title>1</question-title>
> The PDF below contains a scan of the lab books in this example. Somewhere, there is an experiment to show the effect of the knock-down of a gene called CHES-I-like on the feeding behaviour of fruit flies. Try to find this information in the lab book.
> 

Reviewer: ```suggestion
> <question-title></question-title>
> The PDF below contains a scan of the lab books in this example. Somewhere, there is an experiment to show the effect of the knock-down of a gene called CHES-I-like on the feeding behaviour of fruit flies. Try to find this information in the lab book.
```
